addons/vegeta/readme.md

```diff
@@ -0,0 +1,84 @@
+
+# Supported workload type
+Vegeta Trait supports following component types: webservice、worker and cloneset.
```

Comment:
why only support  tree types component?

Reply:
Not only. The three types are easy to use.

Reply:
Other types are alone to users to try.

Reply:
So we can remove this notice? since the `appliesToWorkloads` can take effect to all workloads.

Reply:
I don't think it works for all workloads. We must make it accurate.

Reply:
So `appliesToWorkloads` filed should be align with this notice.